packages/pipcook-app/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "npm run clean && npm run compile",
-    "clean": "rm -rf ./dist && rm -rf tsconfig.tsbuildinfo",
+    "clean": "rm -rf ./dist && rm -rf tsconfig.tsbuildinfo && rm -f package-lock.json",
```

Comment:
This makes a side effect, that we removed a git-managed file, why do we remove the package-lock.json?

Reply:
Ah, this is my mistake. I modify this for test purpose. no good reasons to remove package-lock, I have modified this back